examples/gallery/demos/bokeh/texas_choropleth_example.ipynb

```diff
@@ -38,20 +38,11 @@
     "from bokeh.sampledata.us_counties import data as counties\n",
     "from bokeh.sampledata.unemployment import data as unemployment\n",
     "\n",
-    "counties = {\n",
-    "    code: county for code, county in counties.items() if county[\"state\"] == \"tx\"\n",
-    "}\n",
+    "counties = [dict(county, Unemployment=unemployment[cid])\n",
+    "            for cid, county in counties.items()\n",
+    "            if county[\"state\"] == \"tx\"]\n",
     "\n",
-    "county_xs = [county[\"lons\"] for county in counties.values()]\n",
-    "county_ys = [county[\"lats\"] for county in counties.values()]\n",
-    "\n",
-    "county_names = [county['name'] for county in counties.values()]\n",
-    "county_rates = [unemployment[county_id] for county_id in counties]\n",
-    "\n",
-    "county_polys = {name: hv.Polygons((xs, ys), level=rate, vdims=['Unemployment'])\n",
-    "                for name, xs, ys, rate in zip(county_names, county_xs, county_ys, county_rates)}\n",
-    "\n",
-    "choropleth = hv.NdOverlay(county_polys, kdims=['County'])"
+    "choropleth = hv.Polygons(counties, ['lons', 'lats'], [('detailed name', 'County'), 'Unemployment'])"
```

Comment:
I assume the old code still works and the new approach is just more succinct/flexible/efficient.

Reply:
Right.